eq-publisher/src/getQuestionnaire/queries.js

```diff
@@ -276,6 +277,7 @@ exports.getQuestionnaire = `
             ... on QuestionPage {
               description
               descriptionEnabled
+              pageDescription
```

Comment:
```suggestion
```
This line can be removed  - this file is only used for v2 Publisher, which is no longer maintained